styles.css

```diff
@@ -0,0 +1,99 @@
+/* stylelint-disable no-empty-source */
+
+:root {
+  --primary-color: #6070ff;
+  --primary-color-deep: #2230d2;
+  --base-color: #fff;
+  --grey-text: #344563;
+}
+
+html,
+body {
+  background-color: var(--base-color);
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+}
+
+.container {
+  width: 90%;
+  margin: auto;
+}
+
+#navbar {
+  padding: 18px 0;
+}
+
+#navbar .container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.page_logo {
+  color: var(--primary-color);
+  text-decoration: none;
+  font-size: 18px;
+  font-weight: 700;
+  display: block;
+}
+
+.nav-toggle-btn {
+  background: transparent;
+  outline: none;
+  border: none;
+}
+
+.nav-toggle-btn img {
+  width: 1.5rem;
+}
+
+#headline {
+  position: relative;
+  min-height: 35rem;
+  width: 100%;
+  background: url("./assets/images/header-bg-mobile.png");
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+#headline .container {
+  position: relative;
+  top: 5rem;
+}
+
+.headline-title {
+  font-weight: 700;
+  font-size: 2.6rem;
+  margin: 0;
+}
```

Comment:
As the previous reviewer suggested, the headline title on your design has a different colour from that on the the Figma design (#172B4D). Kindly adjust it.

Your design:
![image](https://user-images.githubusercontent.com/20554583/212023174-7efbecee-0d66-4c96-81fc-e23381deeca3.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/212023194-0b7e0da9-c90d-4800-8747-996d865c32b3.png)